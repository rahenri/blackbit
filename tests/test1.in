new
protover
sd 8
post
e2e4
d2d4
c2c4
g1f3
quit
