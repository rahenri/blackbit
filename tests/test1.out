feature myname="raphael-chess8" ping=1 usermove=1 draw=0 variants="normal" sigint=0 sigterm=0 setboard=1 playother=1 analyze=0 colors=0 done=1
8 -124 0 3214358 e7e5 d1h5 b8c6 f1c4 d8e7 b1c3 c6d4 e1d1
move e7e5
8 -118 0 5827112 d8h4 d4e5 h4e4 d1e2 d7d5 e2e4 d5e4 b1c3
move d8h4
8 -15 0 5946251 h4e4 g1e2 b8c6 b1c3 f8b4 c1d2 b4c3 d2c3
move h4e4
Illegal move: g1f3
